vip_spec.xsd

```diff
@@ -364,13 +359,59 @@
         <xs:element name="zip" type="xs:string"/>
     </xs:all> 
 </xs:complexType> 
-<xs:complexType name="commonIdType">
-    <xs:all>
-        <xs:element name="id_value" type="xs:string" />
-        <xs:element name="id_type" type="commonIdEnum" />
-    </xs:all>
-
+<xs:complexType name="langString">
+    <xs:simpleContent>
+        <xs:extension base="xs:string">
+            <xs:attribute name="lang" type="xs:language" use="required"/>
+        </xs:extension>
+    </xs:simpleContent>
+</xs:complexType>
+<!--
+interText (short for "internationalized text"): a type representing a block
+  of text translated into one or more languages.
+For example, an instance of:
+    <xs:element name="office" type="interText"/>
+could be:
+    <office inter_text_id="office_mayor">
+        <text lang="en">Mayor</text>
+        <text lang="es">Alcalde</text>
+        <text lang="zh">市長</text>
+    </office>
+or (e.g. if only English is available):
+    <office>Mayor</office>
+The optional "inter_text_id" attribute is useful for tracking the string
+source, for example if the translated values came from localized resource
+files.
+-->
+<xs:complexType name="interText">
+    <xs:union>
+        <xs:complexType>
+            <xs:sequence>
+                <xs:element name="text" type="langString" minOccurs="1" maxOccurs="unbounded"/>
+            </xs:sequence>
+            <xs:attribute name="inter_text_id" type="xs:string"/>
+        </xs:complexType>
+        <xs:complexType>
+            <!-- The language defaults to English in this case. -->
+            <xs:simpleContent>
+                <xs:extension base="xs:string">
+                    <xs:attribute name="inter_text_id" type="xs:string"/>
+                </xs:extension>
+            </xs:simpleContent>
+        </xs:complexType>
+    </xs:union>
 </xs:complexType>
+<xs:complexType name="pollingLocationType">
```

Comment:
@pstenbjorn Maybe I'm missing something (quite possible), but does it make sense to put the definition of `polling_location` here (i.e. couldn't this all be defined above)?